bot/cogs/watchchannels/bigbrother.py

```diff
@@ -0,0 +1,118 @@
+import logging
+from collections import ChainMap
+from typing import Union
+
+from discord import Color, Embed, User
+from discord.ext.commands import Context, group
+
+from bot.constants import Channels, Roles, Webhooks
+from bot.decorators import with_role
+from bot.utils.moderation import post_infraction
+from .watchchannel import WatchChannel, proxy_user
+
+log = logging.getLogger(__name__)
+
+
+class BigBrother(WatchChannel):
+    """User monitoring to assist with moderation"""
```

Comment:
Could be more descriptive by specifying their messages are what's monitored.

Reply:
why did we change this in the first place? what's wrong with the old one?

Reply:
Just wrote a new one while I was writing the new class. I can copy the old one, I think it sounds a bit aggressive, though, so that's probably why I didn't copy it.

Reply:
if "NSA-style" is aggressive, I guess I'm done.

Reply:
Hmm, maybe aggressive is not the right word, but it does feel a bit over the top to me. Probably because I'm not really into memeing.